kingpin/actors/base.py

```diff
@@ -308,6 +308,15 @@ def _fill_in_contexts(self, context={}, strict=True):
         # Finally, convert the string back into a dict and store it.
         self._options = json.loads(new_options_string)
 
+    def get_orgchart(self, parent=''):
+        return [
+            [{'v': str(id(self)),
```

Comment:
Maybe for the `'f'` part. `'v'` is used as a unique ID for child-parent relationship. I don't want plain text to identify that because if a description of two actors is the same it would botch the diagram